tests/contrib/print_proxy/test_mapfish_print.py

```diff
@@ -0,0 +1,1470 @@
+# -*- coding: utf-8 -*-
+import pytest
+from pyramid_oereb.contrib.print_proxy.mapfish_print import Renderer
+from tests.renderer import DummyRenderInfo
+
+
+@pytest.fixture()
+def coordinates():
+    return [(((2608901.529, 1261990.655), (2608898.665, 1261991.598), (2608895.798, 1261992.53),
+              (2608892.928, 1261993.452), (2608890.054, 1261994.363), (2608880.256, 1261996.496),
+              (2608873.932, 1261998.379), (2608867.594, 1262000.213), (2608861.242, 1262001.998),
+              (2608854.876, 1262003.733), (2608848.497, 1262005.419), (2608844.698, 1262006.399),
+              (2608796.844, 1261780.919), (2608799.632, 1261780.327), (2608782.789, 1261700.967),
+              (2608782.961, 1261700.722), (2608783.177, 1261700.516), (2608783.43, 1261700.357),
+              (2608783.71, 1261700.251), (2608784.005, 1261700.202), (2608855.37, 1261693.184),
+              (2608870.718, 1261765.538), (2608943.776, 1261750.037), (2608952.331, 1261796.603),
+              (2608961.92, 1261847.725), (2608969.845, 1261889.377), (2608975.711, 1261920.203),
+              (2608913.546, 1261933.412), (2608912.328, 1261934.762), (2608911.162, 1261936.157),
+              (2608910.049, 1261937.595), (2608908.991, 1261939.073), (2608907.988, 1261940.59),
+              (2608907.043, 1261942.144), (2608906.157, 1261943.731), (2608905.331, 1261945.351),
+              (2608904.566, 1261947.0), (2608903.863, 1261948.677), (2608903.223, 1261950.379),
+              (2608902.648, 1261952.104), (2608909.114, 1261982.572), (2608909.22, 1261983.241),
+              (2608909.256, 1261983.918), (2608909.221, 1261984.594), (2608909.115, 1261985.263),
+              (2608908.941, 1261985.918), (2608908.699, 1261986.551), (2608908.393, 1261987.155),
+              (2608908.026, 1261987.724), (2608907.601, 1261988.252), (2608907.124, 1261988.733),
+              (2608901.529, 1261990.655)), )]
+
+
+@pytest.fixture()
+def symbol_ref_base():
+    return 'http://localhost:6543/oereb/image/symbol/'
+
+
+@pytest.fixture()
+def symbol_ref_1():
+    return symbol_ref_base() + 'ContaminatedPublicTransportSites/StaoTyp1'
+
+
+@pytest.fixture()
+def symbol_ref_2():
+    return symbol_ref_base() + 'ContaminatedPublicTransportSites/StaoTyp2'
+
+
+@pytest.fixture()
+def symbol_ref_3():
+    return symbol_ref_base() + 'ContaminatedPublicTransportSites/StaoTyp3'
+
+
+@pytest.fixture()
+def responsible_office_web():
+    return 'http://www.bav.admin.ch/themen/verkehrspolitik/00709/index.html'
+
+
+@pytest.fixture()
+def bav_pdf():
```

Comment:
I'd suggest to link a PDF that is sure not to be removed anytime soon like the certificat for no data for the railway topics of the PLR

Reply:
I see. But actually in this test case, it does not matter, better the test case supplies all the extract data directly (is just tests the way the extract is transformed for mapfish-print).
An additional test case, that actually generates an extract from the db and then calls the conversion code would be cool.
For now, I will merge this first test, because it involves restructuring the mapfish print code and so is a basis for further tests.